sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRestClient.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal partial class TextAnalyticsRestClient
+    {
+        internal HttpMessage CreateHealthStatusNextPageRequest(string nextLink, bool? showStats)
```

Comment:
codegen doesn't create this for us. Looks like it is because of how the swagger defines the pagination. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json#L1695

For next milestone, I need to check with autorest and service teams to see if we can modify it in a way that we can get this for free